src/streamsync/core.py

```diff
@@ -963,6 +963,9 @@ def get_context_data(self, instance_path: InstancePath) -> Dict[str, Any]:
             context[key_variable] = repeater_items[instance_number][0]
             context[value_variable] = repeater_items[instance_number][1]
 
+        if len(instance_path) > 0:
```

Comment:
I should add a type definition (TypedDict) on context to allow autocompletion in the IDE